<!-- For backwards compatibility with the old driver files
                Will be DELETED in the future              -->

<launch>
  <group ns="leap_motion">
    <node pkg="leap_motion" type="leap_camera" name="leap_camera" output="screen"/>
    <node pkg="leap_motion" type="leap_hands" name="leap_hands" output="screen"/>
  </group>
</launch>
